app/controllers/idv/socure/document_capture_controller.rb

```diff
@@ -65,6 +67,22 @@ def update
         # Not used in standard flow, here for data consistency with hybrid flow.
         document_capture_session.confirm_ocr
 
+        # If the stored_result is nil, the job fetching the results has not completed.
+        if stored_result.nil?
+          analytics.idv_doc_auth_document_capture_polling_wait_visited(**analytics_arguments)
+          if wait_timed_out?
+            # flash[:error] = I18n.t('errors.doc_auth.polling_timeout')
+            # TODO: redirect to try again page LG-14873/14952/15059
+            render plain: 'Technical difficulties!!!', status: :ok
```

Comment:
while this will become a redirect very shortly ... would it be easy to add a simple spec here?

Reply:
Done in 20ee038f6e